mllib/src/main/scala/org/apache/spark/mllib/linalg/Matrices.scala

```diff
@@ -87,6 +87,9 @@ sealed trait Matrix extends Serializable {
   /** A human readable representation of the matrix */
   override def toString: String = toBreeze.toString()
 
+  /** A human readable representation of the matrix with specific lines and width */
```

Comment:
It is not clear to me that what `maxWidth` means. It could be max number of cols or max number of chars. I think it is better to rename the arguments to `maxRow` and `maxCols`.